libs/api/domains/directorate-of-labour/src/dto/getParentalLeavesEstimatedPaymentPlan.input.ts

```diff
@@ -11,9 +11,9 @@ class Period {
   @IsString()
   to!: string
 
-  @Field(() => Float)
+  @Field(() => String)
   @IsNumber()
-  ratio!: number // Ratio of usage in period.
+  ratio!: string // Ratio of usage in period.
```

Comment:
Would be good to document what kind of values this field supports. The name "ratio" is not great. Not sure if we want to "clean" that up in our schema.